content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +202,62 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
```

Comment:
Proposing "Before we continue with the following recommendations, make sure you understand the options by checking [Creating an update graph with OLM][upgrade-graph]."